modules/core/src/main/java/org/apache/ignite/internal/pagemem/wal/record/ConsistentCutFinishRecord.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.pagemem.wal.record;
+
+import java.util.Set;
+import java.util.UUID;
+import org.apache.ignite.internal.processors.cache.persistence.tree.io.CacheVersionIO;
+import org.apache.ignite.internal.processors.cache.version.GridCacheVersion;
+import org.apache.ignite.internal.util.tostring.GridToStringInclude;
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+/**
+ * This record is written to WAL after Consistent Cut finished on a baseline node.
+ * <p>
+ * It guarantees that the BEFORE side consists of:
+ * 1. Transactions committed before {@link ConsistentCutStartRecord} and weren't included into {@link #after()}.
+ * 2. Transactions committed between {@link ConsistentCutStartRecord} and {@link ConsistentCutFinishRecord}
+ *    and were included into {@link #before()}.
+ * <p>
+ * It guarantees that the AFTER side consists of:
+ * 1. Transactions committed before {@link ConsistentCutStartRecord} and were included into {@link #after()}.
+ * 2. Transactions committed between {@link ConsistentCutStartRecord} and {@link ConsistentCutFinishRecord}
+ *    and weren't included into {@link #before()}.
+ */
+public class ConsistentCutFinishRecord extends WALRecord {
+    /** Consistent Cut ID. */
+    @GridToStringInclude
+    private final UUID cutId;
```

Comment:
The same field inside `ConsistentCut` class names `id`. 
Let's rename this field to `id` also.